src/core/mixins/MediaInfo.ts

```diff
@@ -224,6 +224,29 @@ export default class MediaInfo {
     }, url_params);
   }
 
+  /**
+   * Adds video to the watch history.
+   */
+  async updateWatchHistoryTime(playedSeconds: number, client_name = Constants.CLIENTS.WEB.NAME, client_version = Constants.CLIENTS.WEB.VERSION, replacement = 'https://www.'): Promise<Response> {
```

Comment:
To fix the compatibility with the main branch please add the types here.
```suggestion
  async updateWatchHistoryTime(playedSeconds: number, client_name: string = Constants.CLIENTS.WEB.NAME, client_version: string = Constants.CLIENTS.WEB.VERSION, replacement = 'https://www.'): Promise<Response> {
```